solarwinds_apm/apm_config.py

```diff
@@ -272,6 +276,66 @@ def calculate_is_lambda(cls) -> bool:
             return True
         return False
 
+    @classmethod
+    def calculate_logs_enabled(
+        cls,
+        is_legacy: bool = False,
+        cnf_dict: dict = None,
+    ) -> bool:
+        """Return if export of instrumentor logs telemetry enabled.
+        Invalid boolean values are ignored.
+        Order of precedence: Environment Variable > config file > default.
+        Default is True is not legacy, False if legacy.
+        Optional cnf_dict is presumably already from a config file, else a call
+        to get_cnf_dict() is made for a fresh read."""
+        if is_legacy:
+            logs_enabled = False
+        else:
+            logs_enabled = True
```

Comment:
I vaguely remember having a conversation about not turning on logs export by default based on pricing uncertainty (as in to avoid customers getting a surprise bill because they're now sending in a bunch of logs), has that been settled ?

Reply:
Ah good catch. No, I did not come to a conclusion on this. I will check and get back!

Reply:
That's right, we don't want to enable logs export by default, it should be explicitly opted into by the customer.

Reply:
Thank you both! Updated in [34c8513](https://github.com/solarwinds/apm-python/pull/508/commits/34c8513eaab62ac82744ffe32e5728b167c6c60f), and PR description too.